wp-content/plugins/core/src/Panels/Types/MediaText.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Tribe\Project\Panels\Types;
+
+use ModularContent\Fields;
+
+class MediaText extends Panel_Type_Config {
+
+	const NAME = 'mediatext';
+
+	const FIELD_TITLE          = 'title';
+	const FIELD_DESCRIPTION    = 'description';
+	const FIELD_IMAGE          = 'image';
+	const FIELD_VIDEO          = 'video';
+	const FIELD_CTA            = 'cta';
+	const FIELD_MEDIA_TYPE	   = 'media_type';
+	const FIELD_MEDIA_POSITION = 'media_position';
+	const FIELD_LAYOUT         = 'layout';
+
+	const OPTION_MEDIA_TYPE_IMAGE   = 'option_media_type_image';
```

Comment:
In seeing how this plays out, I like the namespacing of the strings less. We already get verbosity with the constant names. If we simplify these strings to be like: `right`, `left`, etc. then we also benefit from not having to do conditionals like this: https://github.com/moderntribe/square-one/pull/355/files#diff-39f42d7c872f2cdc85b67560c572c96eR256

Reply:
I concur. In ACF it's more necessary since those field names can conflict, but in the Panels context keeping the actual values generic should be just fine and dandy!

Reply:
Sounds good!